app/views/admin/items/_form.html.erb

```diff
@@ -4,7 +4,7 @@
   <%= form.autocomplete_text_field :name, path: admin_ui_names_path(format: :json), autofocus: true,
       hint: "Generic name for the tool; e.g. Impact driver, Orchard ladder, Flathead screwdriver" %>
 
-  <%= form.text_field :other_names,
+  <%= form.text_field :other_names_and_keywords,
```

Comment:
I think we need to change just the label that is displayed like this:

```suggestion
  <%= form.text_field :other_names, label: "other_names_and_keywords",
```

Otherwise, the form builder will assume that the field is called `:other_names_and_keywords` and it will fail to save when the user submits the form because the name of the field, what's allowed in `item_params` in the controller, and the database column name need to all match.

Reply:
oo didn't know that was all lumped together in rails forms, thanks!

Reply:
made the change and tested the form to make sure it still worked 🙂